rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -281,6 +294,12 @@ BuildRequires: curl-devel
 Requires: curl
 %endif
 
+## JQ dependencies
+%if "%{use_system_jq}" == "true"
+BuildRequires: jq-devel
```

Comment:
What is it used for?

Reply:
It is what I thought was the fix for @peckpeck comment: 
> You must also declare jq as a dependency here when system_jq is true
> And declafe jq-devel as a build-dependency too

Reply:
do we build something relying on jq? CFEngine only uses the binary.

Reply:
i thought it was using the library

Reply:
so : 
- no JQ_ARGS
- no change to cfengine configure
- no change to build-depends

Reply:
removed some things, hopefully the one I had to